Assets/Scripts/Bullet.cs

```diff
@@ -0,0 +1,17 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class Bullet : MonoBehaviour
+{
+    private void OnTriggerEnter2D(Collider2D other) 
+    {
+        Debug.Log("Hit something");
+        if (other.gameObject.CompareTag("Player"))
```

Comment:
A bullet should have an owner/faction tag as well, so that players can't shoot themselves, and later when we add player shooting, enemies can't shoot themselves either.